@here/harp-mapview/lib/geometry/TileGeometryCreator.ts

```diff
@@ -308,6 +348,29 @@ export class TileGeometryCreator {
         // enabledKinds and disabledKinds, in which case they are flagged. The disabledKinds can be
         // ignored hereafter.
         this.initDecodedTile(decodedTile, enabledKinds, disabledKinds);
+
+        // compile the dynamic expressions.
+        const exprPool = tile.dataSource.exprPool;
+        decodedTile.techniques.forEach((technique: any) => {
+            for (const propertyName in technique) {
+                if (!technique.hasOwnProperty(propertyName)) {
+                    continue;
+                }
+                const value = technique[propertyName];
+                if (isJsonExpr(value) && propertyName !== "kind") {
+                    // "kind" is reserved.
+                    try {
+                        let expr = Expr.fromJSON(value);
+                        if (expr.dependencies().volatile !== true) {
+                            expr = new MemoCall(Expr.fromJSON(value));
+                        }
+                        technique[propertyName] = expr.intern(exprPool);
+                    } catch (error) {
+                        logger.error("#TileGeometryCreator: Failed to compile expression:", error);
```

Comment:
This logger instance already prefixes all logs with "TileGeometryCreator: "...

Reply:
removed